src/utils/constants.ts

```diff
@@ -58,6 +58,15 @@ export const GWEI: BN = WEI.mul(new BN(1000000000))
 export const AVAXGWEI: BN = NANOAVAX.clone()
 export const AVAXSTAKECAP: BN = ONEAVAX.mul(new BN(3000000))
 
+export const KopernikusID = "1002"
+export const ColumbusID = "1001"
+export const CaminoID = "1000"
+
+export const BERLINPHASETIME: Map<string, Date> = new Map([
+  [KopernikusID, new Date("2024-04-13T00:00:00Z")],
+  [ColumbusID, new Date("2024-08-07T00:00:00Z")],
+  [CaminoID, new Date("2024-08-17T00:00:00Z")]
```

Comment:
for Berlin, I think this date is too early.

Reply:
@mo-c4t  Do you know where it is possible to check the dates? The only source I have is https://github.com/chain4travel/caminogo/blob/dev/version/constants.go#L184

Reply:
We still don't have final dates. but I think you can put the dates more into the future from Columbus and Camino until we have final dates. what do you think @evlekht ?

Reply:
yes, ofc, put something temporal